devops/docker/idea-image/ideaIC-2022.2.2/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM cimg/openjdk:11.0
+
+LABEL maintainer="Checkstyle organization https://github.com/orgs/checkstyle/people"
+
```

Comment:
Do we need this?
We usually keep all files not related to certain people

Reply:
removed